### PR TITLE
cli/release: adding 'release update' to allow tweaking existing releases

### LINKED ISSUE #1535: cli: Add `flynn release update`
https://botbot.me/freenode/flynn/msg/39273010/ discusses how to update a release. (Use case is to make an app that publishes memory stats to CloudWatch "omni", to enable autoscaling based on memory.)

https://github.com/flynn/flynn/blob/master/cli/release.go#L119-L151 is the add statement we can mimic.

### PR DESCRIPTION
**Pull request by temujin9**
Signed-off-by: Nathaniel Eliot <nathaniel.eliot@bazaarvoice.com>

Fixes #1535

### CODE REVIEW

**Comment by titanous** [comment]
- The commit message prefix should be `cli:` not `cli/release:`
- The code needs a `gofmt -s -w`.

**Comment by temujin9** [comment]
https://ci.flynn.io/builds/20160120020445-6ffdea3e failed for no obvious reason: I suspect something is flaky with CI again. Trying again, but my luck has been poor thus far . . .

**Comment by temujin9** [comment]
https://ci.flynn.io/builds/20160120043412-81d544ea didn't fare any better. Giving up for tonight.

**Comment by temujin9** [comment]
https://ci.flynn.io/builds/20160120145313-b0fee0bc also bombed, for similar reasons that don't appear to have anything to do with this. I'm done trying until those have been resolved.

**Comment by temujin9** [comment]
Added tests: https://ci.flynn.io/builds/20160121224149-7a787677

**Comment by temujin9** [comment]
Well, it builds cleanly on my local machine, at any rate. @titanous - let me know if you need any further revisions.

File: cli/release.go [hunk 1]
```diff
@@ -183,3 +207,92 @@ func runReleaseAddDocker(args *docopt.Args, client *controller.Client) error {
...
+			if len(procUpdate.Ports) > 0 {
+				procRelease.Ports = procUpdate.Ports
+			}
+			if procUpdate.Data == true {
+				procRelease.Data = true
+			}
+			if procUpdate.Omni == true {
+				procRelease.Omni = true
+			}
+			if procUpdate.HostNetwork == true {
```
Comment (titanous) [inline cli/release.go]: No need for `== true`, `bool` values can be used without comparison.

Comment (temujin9) [inline cli/release.go]: Fair enough. I'll let this build finish, and then push that change.

**Comment by temujin9** [comment]
Okay, code should be complete, and all tests pass locally. Builds keep failing for unrelated reasons. Let me know if more tweaks are needed.

**Comment by titanous** [comment]
LGTM

**Pull request merged**